web/src/main/java/org/fao/geonet/monitor/link/LinkMonitorService.java

```diff
@@ -0,0 +1,142 @@
+package org.fao.geonet.monitor.link;
+
+import jeeves.interfaces.Logger;
+import jeeves.server.resources.ResourceManager;
+import jeeves.utils.Log;
+import org.apache.lucene.document.DocumentStoredFieldVisitor;
+import org.fao.geonet.GeonetContext;
+import org.fao.geonet.constants.Geonet;
+import org.fao.geonet.kernel.search.IndexAndTaxonomy;
+import org.fao.geonet.kernel.search.index.GeonetworkMultiReader;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class LinkMonitorService extends LinkMonitorInterface {
+
+    public enum Status {
```

Comment:
This enum isn't used in this class.  Would it be better off somewhere else (`LinkInfo` perhaps)?